api/v1/models/MailingList.js

```diff
@@ -0,0 +1,27 @@
+var Model = require('./Model');
+var User = require('./User');
+var MailingListUser = require('./MailingListUser');
+
+var MailingList = Model.extend({
+	tableName: 'mailing_lists',
+	idAttribute: 'id'
+});
+
```

Comment:
We could add a unique constraint to the `name` field for mailing lists. If you're talking about avoiding the addition of one email to a list twice, we could probably check that in `addToList` (which will be moved here)